doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -444,7 +444,7 @@ cygnus-ngsi.sinks.cartodb-sink.enable_grouping = false
 cygnus-ngsi.sinks.cartodb-sink.enable_name_mappings = false
 cygnus-ngsi.sinks.cartodb-sink.enable_lowercase = false
 cygnus-ngsi.sinks.cartodb-sink.keys_conf_file = /usr/cygnus/conf/cartodb_keys.conf
-cygnus-ngsi.sinks.cartodb-sink.flip_coordinates = true
+cygnus-ngsi.sinks.cartodb-sink.swap_coordinates = true
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1329/commits/423f020dace40fa1f12f7ec25758fe3fcc3aee13